configure

```diff
@@ -1129,6 +1129,30 @@ EOF
     fi
 }
 
+
+check_acle_intrinsics() {
```

Comment:
This function doesn't really check for ACLE intrinsics. It checks for availability of ACLE architecture compiler flags. So maybe rename `check_acle_compiler_flag`?

Reply:
Neither does `check_neon_intrinsics()` in configure... We would need to rename both and that would require another PR.

Reply:
Why not fix your commit and then the other PR can fix the other one?

Reply:
You could even add another commit to this PR that fixed it. I think it would get accepted.